vrecord

```diff
@@ -572,6 +572,71 @@ _writeingestlog(){
     fi
 }
 
+# create a png of qc data graphs for quick assessment
+_qcgraphimage(){
+    # get audio data
+    qctools_CONFIGFILE_A="$(/usr/bin/mktemp /tmp/astats_A_XXXXXXXXX).csv"
+    gzcat "$1" | perl -nle 'print if not m{lavfi.(?!astats.1.Min_level|astats.1.Max_level|astats.1.Peak_level)}' | xml select -t -m "//ffprobe:ffprobe/frames/frame[@media_type='audio']" \
```

Comment:
noting that this uses the first audio channel rather than the overall audio. Perhaps `astats.Overall` would be better throughout rather than `astats.1`.